www/changelog.html

```diff
@@ -0,0 +1,138 @@
+            <img src="/images/run_tag_diff.png" alt="Diff two different tagged versions of the same run" />
+          </a>
+        </div>
+      </div>
+    </div>
+
+    <div class="intro-row">
+      <div class="content">
+        <div class="intro-col intro-col-left col-md-2">
+          <h3>Generate index.html file by using <i>PlistToHTML</i></h3>
+          <p>
+            Creating an <a href="https://github.com/Ericsson/codechecker/blob/master/docs/usage.md#step-3">
+            <span class="code">index.html</span></a> file that lists
+            the found reports which links to the local html pages.<br/>
+            Usage: <span class="code">CodeChecker parse ./reports -e html -o ./reports_html</span>
+          </p>
+        </div>
+        <div class="intro-col intro-col-right col-md-2">
+          <a href="/images/plist_to_html_index.png" data-dojo-type="dojox.image.Lightbox" title="Generate index.html file by using PlistToHTML">
+            <img src="/images/plist_to_html_index.png" alt="Generate index.html file by using PlistToHTML" />
+          </a>
+        </div>
+      </div>
+    </div>
+
+    <div class="intro-row">
+      <div class="content">
+        <div class="intro-col intro-col-left">
+          <h3>Separate filter options for cmd line</h3>
+          <p>
+          Create separate <a href="https://github.com/Ericsson/codechecker/blob/master/docs/user_guide.md#cmd">filter options</a>
+          for <span class="code">CodeChecker cmd results</span> and
+          <span class="code">CodeChecker cmd diff</span> command line options to
+          be more consistent with the UI.<br/>
+          Examples:
+          <ul>
+            <li>
+                <span class="code">CodeChecker cmd results my_run --severity
+                    critical high medium --file "/home/username/my_project/*"</span>
+            </li>
+            <li>
+                <span class="code">CodeChecker cmd diff -b run1 -n run2
+                  --unresolved --severity high medium --review-status
+                  unreviewed confirmed</span>
+            </li>
+          </ul>
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="intro-row">
+      <div class="content">
+        <div class="intro-col intro-col-left">
+          <h3>Enable passwordless token based authentication</h3>
+          <p>
+            Command line clients can authenticate itself using the
+            username/password stored in the
+            <span class="code">.codechecker.passwords.json</span>. It is
+            obviously not a good idea to store passwords in text files.
+            Instead of this the user is able to generate a
+            <a href="https://github.com/Ericsson/codechecker/blob/master/docs/authentication.md#personal-access-token">token</a>
+            from command line, that can be used to authenticate in the name of
+            his/her name.
```

Comment:
I think it would be nice to mention here that the tokens can be generated in the command line now and give some short example how to create and list the tokens `CodeChecker cmd token new` and `CodeChecker cmd token list`